client/packages/common/src/intl/locales/en/distribution.json

```diff
@@ -11,6 +11,7 @@
   "button.new-shipment": "New Shipment",
   "button.select-a-color": "Select a colour",
   "button.stop": "Stop",
+  "error.failed-to-create-return": "Failed to create return!",
```

Comment:
Wow just realised that we duplicate strings for distribution and replenishment

Reply:
I've added a small comments here: https://github.com/msupply-foundation/open-msupply/issues/1266#issuecomment-1957995627

Reply:
Ah awesome thanks Andrei, yeah would be great to consolidate those!